p2p/src/peer_manager/tests/ban.rs

```diff
@@ -60,20 +60,12 @@ where
     )
     .await;
     let peer_id = peer_info.peer_id;
-    pm2.accept_inbound_connection(address, peer_info, None).unwrap();
+    pm2.accept_inbound_connection(address, peer_info, None).await.unwrap();
 
     assert_eq!(pm2.adjust_peer_score(peer_id, 1000).await, Ok(()));
```

Comment:
I just noticed this number... I know it's not from you, Pavel. But why is this 1000? Seems arbitrary. Please throw in a TODO around it to check how to do this properly.

If I have to guess, someone did this like that because they want to just ban the peer no matter what. This doesn't seem to be the correct approach.